src/components/navigation/NavigationContent.tsx

```diff
@@ -0,0 +1,60 @@
+import Link from 'next/link';
+import { ListGroup } from 'react-bootstrap';
+import { useTranslations } from 'next-intl';
+
+// TODO: Replace with actual data from configuration
+export default function NavigationContent({ className }: { className?: string }) {
+  const t = useTranslations('NavigationContent');
+  return (
+    <ListGroup variant="flush" className={className}>
+      <ListGroup.Item>
+        <Link href="#" className="nav-link">
+          <i className="bi bi-house-door-fill" /> Home
+        </Link>
+      </ListGroup.Item>
+      <ListGroup.Item>
+        <Link href="#" className="nav-link">
+          <i className="bi bi-search" /> {t('search')}
+        </Link>
+      </ListGroup.Item>
+      <ListGroup.Item>
+        <Link href="#" className="nav-link">
+          <i className="bi bi-bar-chart-fill" /> Dashboards
+        </Link>
+        <div className="d-flex flex-column">
+          <Link href="#" className="link-secondary link-underline-opacity-0 m-1 ms-3">
+            <i className="bi bi-book-half" /> Bevölkerung
```

Comment:
Are the icons still WIP? because I believe one of these might look nicer, and fit the theme of a dashboard better: https://icons.getbootstrap.com/?q=chart

Reply:
Yeah, the icons are WIP. Probably, we will make the exact icons a configuration option